### PR TITLE
Explain how to use _TOKEN_ to disable BO token security

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Explain how to use `_TOKEN_` to disable BO token security
| Fixed ticket? |

### CODE REVIEW

File: src/content/1.7/basics/installation/configuration.md [hunk 1]
```diff
@@ -93,3 +93,10 @@ When in production mode, `$smarty->force_compile` must be set to `false`, as it
 On the other hand, when editing a `.tpl` file, you must delete the `/tools/smarty/compile` folder (except the `index.php` file) in order to see your changes applied.
 
 Note that this setting can be made directly from the back office, in the “Performance” page under the “Advanced parameters” menu.
+
+## Disable the Back-Office token protection
+
+Back-Office pages require the use of a token. If needed, this protection can be disabled using an environment variable:
```
Comment (PierreRambaud) [inline src/content/1.7/basics/installation/configuration.md:99]:
```suggestion
Back-Office pages require the use of a token. If needed, this protection can be disabled using an environment variable:

### Apache with mod_headers
```

File: src/content/1.7/basics/installation/configuration.md [hunk 2]
```diff
@@ -93,3 +93,10 @@ When in production mode, `$smarty->force_compile` must be set to `false`, as it
...
+```bash
+SetEnv _TOKEN_ disabled
+```
```
Comment (PierreRambaud) [inline src/content/1.7/basics/installation/configuration.md:102]:
```suggestion
```
### Nginx with ngx_http_headers_module
```bash
add_header _TOKEN_ disabled;
```
```

Comment (PierreRambaud) [reply src/content/1.7/basics/installation/configuration.md:102]: Can add ` in the suggestion :sweat:

**Comment by matks** [comment]
@PierreRambaud I think I got the idea 😉 I applied the feedback

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @matks

**Comment by mickaelandrieu** [comment]
Hi,

it was already documented but in another part of the docs => https://devdocs.prestashop.com/1.7/scale/benchmark/back-office/#disable-the-token

Regards,

**Pull request merged**